src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -445,7 +471,46 @@ Whether Harvesting (OAI) service is enabled
          */
         
         ArchiverClassName,
+        /**
+         * Custom settings for each archiver. See list below.
+         */
         ArchiverSettings,
+        /**
```

Comment:
FWIW: These archiver settings were originally not listed in the SettingsServiceBean because the archivers were meant to plugable, with new archivers able to add their own settings as needed.

Reply:
Not an immediate problem. We haven't had new archivers after the initial set (though perhaps OCFL someday), but I think longer term we do want plugins (in general, exporters, pid providers, etc. not just archivers) to be able to define their own settings.